ariadne/cardano/src/Ariadne/Config/CLI.hs

```diff
@@ -70,9 +70,11 @@ data CLI_ConfigurationOptions = CLI_ConfigurationOptions
     } deriving (Eq, Show, Generic)
 
 data CLI_WalletConfig = CLI_WalletConfig
-    { cli_wcEntropySize :: !(Maybe Byte)
-    , cli_wcKeyfilePath :: !(Maybe FilePath)
-    , cli_wcAcidDBPath  :: !(Maybe FilePath)
+    { cli_wcEntropySize       :: !(Maybe Byte)
+    , cli_wcKeyfilePath       :: !(Maybe FilePath)
+    , cli_wcAcidDBPath        :: !(Maybe FilePath)
+    , cli_wcNumStoredArchives :: !(Maybe Int)
+    , cli_wcDBCleanupPerion   :: !(Maybe Int)
```

Comment:
Why are you using `Int` instead of `o-clock`'s `Second`?

Reply:
I afraid that using unbounded types in config could cause out of memory if someone accidentally will write a too big number. `Time Second` type is actually a `Ratio Natural` and user can accidentally write a too big number (or it can be parsed in a wrong way). 
Am I wrong?  Am I wrong?

Reply:
Well, I don't think someone will ever _accidentally_ write a number that takes hundreds of megabytes.
And anyway, you can read it as `Int` and convert to `Time Second` in the parser. The earlier you do this conversion, the better.